{"schema_version":"1.6.0","id":"OESA-2022-1485","modified":"2022-01-07T11:31:14Z","published":"2022-01-07T11:31:14Z","related":["CVE-2021-41496"],"summary":"numpy security update","details":"A fast multidimensional array facility for Python.\r\n\r\nSecurity Fix(es):\r\n\r\nBuffer overflow in the array_from_pyobj function of fortranobject.c in NumPy &lt; 1.19, which allows attackers to conduct a Denial of Service attacks by carefully constructing an array with negative values.(CVE-2021-41496)","affected":[{"package":{"ecosystem":"openEuler-20.03-LTS-SP1","name":"numpy","purl":"pkg:rpm/openEuler/numpy&distro=openEuler-20.03-LTS-SP1"},"ranges":[{"type":"ECOSYSTEM","events":[{"introduced":"0","fixed":"1.16.5-4.oe1"}]}],"ecosystem_specific":{"aarch64":["python3-numpy-f2py-1.16.5-4.oe1.aarch64.rpm","python2-numpy-1.16.5-4.oe1.aarch64.rpm","numpy-debugsource-1.16.5-4.oe1.aarch64.rpm","python2-numpy-f2py-1.16.5-4.oe1.aarch64.rpm","python3-numpy-1.16.5-4.oe1.aarch64.rpm","numpy-debuginfo-1.16.5-4.oe1.aarch64.rpm"],"src":["numpy-1.16.5-4.oe1.src.rpm"],"x86_64":["python2-numpy-f2py-1.16.5-4.oe1.x86_64.rpm","python3-numpy-1.16.5-4.oe1.x86_64.rpm","numpy-debugsource-1.16.5-4.oe1.x86_64.rpm","numpy-debuginfo-1.16.5-4.oe1.x86_64.rpm","python2-numpy-1.16.5-4.oe1.x86_64.rpm","python3-numpy-f2py-1.16.5-4.oe1.x86_64.rpm"]}},{"package":{"ecosystem":"openEuler-20.03-LTS-SP2","name":"numpy","purl":"pkg:rpm/openEuler/numpy&distro=openEuler-20.03-LTS-SP2"},"ranges":[{"type":"ECOSYSTEM","events":[{"introduced":"0","fixed":"1.16.5-4.oe1"}]}],"ecosystem_specific":{"aarch64":["python3-numpy-f2py-1.16.5-4.oe1.aarch64.rpm","numpy-debuginfo-1.16.5-4.oe1.aarch64.rpm","python3-numpy-1.16.5-4.oe1.aarch64.rpm","python2-numpy-1.16.5-4.oe1.aarch64.rpm","python2-numpy-f2py-1.16.5-4.oe1.aarch64.rpm","numpy-debugsource-1.16.5-4.oe1.aarch64.rpm"],"src":["numpy-1.16.5-4.oe1.src.rpm"],"x86_64":["python3-numpy-1.16.5-4.oe1.x86_64.rpm","python2-numpy-f2py-1.16.5-4.oe1.x86_64.rpm","numpy-debugsource-1.16.5-4.oe1.x86_64.rpm","python2-numpy-1.16.5-4.oe1.x86_64.rpm","numpy-debuginfo-1.16.5-4.oe1.x86_64.rpm","python3-numpy-f2py-1.16.5-4.oe1.x86_64.rpm"]}},{"package":{"ecosystem":"openEuler-20.03-LTS-SP3","name":"numpy","purl":"pkg:rpm/openEuler/numpy&distro=openEuler-20.03-LTS-SP3"},"ranges":[{"type":"ECOSYSTEM","events":[{"introduced":"0","fixed":"1.16.5-4.oe1"}]}],"ecosystem_specific":{"aarch64":["python3-numpy-f2py-1.16.5-4.oe1.aarch64.rpm","numpy-debuginfo-1.16.5-4.oe1.aarch64.rpm","python3-numpy-1.16.5-4.oe1.aarch64.rpm","python2-numpy-1.16.5-4.oe1.aarch64.rpm","python2-numpy-f2py-1.16.5-4.oe1.aarch64.rpm","numpy-debugsource-1.16.5-4.oe1.aarch64.rpm"],"src":["numpy-1.16.5-4.oe1.src.rpm"],"x86_64":["python3-numpy-1.16.5-4.oe1.x86_64.rpm","python2-numpy-f2py-1.16.5-4.oe1.x86_64.rpm","numpy-debugsource-1.16.5-4.oe1.x86_64.rpm","python2-numpy-1.16.5-4.oe1.x86_64.rpm","numpy-debuginfo-1.16.5-4.oe1.x86_64.rpm","python3-numpy-f2py-1.16.5-4.oe1.x86_64.rpm"]}}],"references":[{"type":"ADVISORY","url":"https://www.openeuler.org/en/security/safety-bulletin/detail.html?id=openEuler-SA-2022-1485"},{"type":"ADVISORY","url":"https://nvd.nist.gov/vuln/detail/CVE-2021-41496"}],"database_specific":{"severity":"High"}}
